src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -36,7 +36,7 @@ export default () => `
 							<div class="flex">
 								<div class="icon"></div>
 								<div class="title">
-									<h1 class="caption"></h1>
+									<h1 class="caption">${escape(localize('welcomePage.caption', "Azure Data Studio"))}</h1>
```

Comment:
why are we making this change?

Reply:
@yualan

Reply:
Hi Alan, there were some reports about some formatting issues with an extended name. In case there are other versions that may change the name in some way (sometimes this happens on macOS for Azure Data Studio 1), it may be worth de-risking the release by hardcoding Azure Data Studio.

![image](https://user-images.githubusercontent.com/30873802/78949478-8391e280-7a80-11ea-874d-f14b16775523.png)

Reply:
is this the view even in maximized mode? i think the right fix here is to set this h1 not to wrap, if user reduce the size of the ADS to certain point, even 'Azure Data Studio' will wrap

Reply:
Hi Alan, for the initial launch of the welcome page, I would prefer that we take the safer approach and have all builds just show Azure Data Studio in the welcome page. We can take customer feedback if they want to see what build they are on there, but would rather just always show Azure Data Studio.